### DISCUSSION TITLE
Reducing number of nodes on mesh

### DISCUSSION THREAD

**Discussion by sdantu2** [neperfepx/neper#879] [Q&A] [answered]
As the title implies, I need help of reducing the number of nodes below 1000 on a mesh as that is the maximum number of nodes that I can have on a mesh for simulation in Abaqus student edition. I have tried to use both -cl and -rcl commands to try to reduce the number of nodes, but I cannot get the node number to reduce below roughly 18000. 

This is what I used for the tessellation: neper -T -n 100 -domain "cylinder(3,3)" -morpho "diameq:lognormal(0.150, 0.05), 1-sphericity:lognormal(0.145, 0.03) -o sample1

This is what I used for meshing: neper -M sample1.tess -elttype tet *-cl (variable values)* -> experimented with -rcl as well

In the meshing, I have used -cl and -rcl individually with variable values, however, I cannot get the number of nodes to what I require. I am not sure if there are limitations due to the way that I defined my tessellation or if I am just simply missing a command. Any help would be appreciated.

**Comment by rquey** [comment, marked as answer]
100 grains -> ~700 vertices, so it is almost hopeless already.

What I would advice you it to use "mapped meshing", using `-rcl hex`, so that you can control (and minimize) and the number of nodes in the model.

```
neper -T -n 100 -o n100
neper -M n100.tess -elt hex -cl 1/12
```
will give you 8281 nodes, and it will look like this:
![d](https://github.com/neperfepx/neper/assets/2659376/02925823-8840-4713-b5a5-f07e10a146c2)
